tests/module-setup.sh

```diff
@@ -58,7 +58,7 @@ elif [[ $MACHINE_ID = stampede ]] ; then
     fi
     module purge
 
-elif [[ $MACHINE_ID = gaea ]] ; then
+elif [[ $MACHINE_ID = gaea* ]] ; then
```

Comment:
also separate this out into a gaea_c5 MACHINE_ID as the lmod is different

Reply:
done